win32ss/user/ntuser/display.c

```diff
@@ -146,6 +146,21 @@ InitDisplayDriver(
     return pGraphicsDevice;
 }
 
+BOOLEAN
+NTAPI
+UserGetBaseVideo(VOID)
+{
+    return gbBaseVideo;
+}
+
+VOID
+NTAPI
+UserSetBaseVideo(
```

Comment:
Is it of actual use to have those getter/setter functions for gbBaseVideo? I don't recall seeing the other globals being manipulated that way. Do you intend to add more to those functions later on?

Extra remark: If those functions are not meant to be exported by win32k and are not meant to be "callbacks", you can remove the "NTAPI" thing.

Reply:
Removed functions.